fastlite/core.py

```diff
@@ -155,9 +155,8 @@ def create(
     strict=False,  # Apply STRICT mode to table
 ):
     "Create table from `cls`, default name to snake-case version of class name"
-    flexiclass(cls)
```

Comment:
Why is this moved?

Reply:
Flexiclass doesn't change data in-place, so this line is superfluous.

The types dict is constructed after the name is extracted, and __name__ isn't limited to just dataclasses.

Reply:
But flexiclass *does* change it in place! We want `cls` to be modified to create a flexiclass :D